roles/qe-server/README.rst

```diff
@@ -3,3 +3,78 @@
 ================
 
 This role configures the machine for execution of usmqe integration tests.
+
```

Comment:
It seems like this and the following files are part of this pull request by mistake as it is about _Xvfb_ server and other stuff and not about _ceph-ansible_.